mono_repo/lib/src/commands/github/github_yaml.dart

```diff
@@ -134,11 +138,21 @@ Iterable<MapEntry<String, Map<String, dynamic>>> _listJobs(
 
     if (mergeStages.contains(first.job.stageName)) {
       final packages = entry.value.map((t) => t.job.package).toList();
-      yield _jobEntry(first.jobYaml(packages));
+      yield jobEntry(first.jobYaml(packages));
     } else {
-      yield* entry.value.map(
-        (jobEntry) => _jobEntry(jobEntry.jobYaml()),
-      );
+      yield* entry.value.map((e) => jobEntry(e.jobYaml()));
+    }
+  }
+
+  // Generate the jobs that run on completion of all other jobs, by adding the
+  // appropriate `needs` config to each.
+  if (onCompletionJobs != null && onCompletionJobs.isNotEmpty) {
```

Comment:
would be good to add a test just for this – it's pretty easy in the `github_command_test` file

Reply:
what would you like to test specifically? the readme test does effectively cover some of it.